website/intro/vs/boto.html.markdown

```diff
@@ -3,7 +3,7 @@ layout: "intro"
 page_title: "Terraform vs. Boto, Fog, etc."
 sidebar_current: "vs-other-boto"
 description: |-
-  Libraries like Boto, Fog, etc. are used to provide native access to cloud providers and services by using their APIs. Some libraries are focused on specific clouds, while others attempt to bridge them all and mask the semantic differences. Using a client library only provides low-level access to APIs, requiring application developers to create their own tooling to build and manage their infrastructure.
+  How Terraform compares to cloud provider client libraries. 
```

Comment:
We should keep specific names of technologies that the page covers in the descriptions so that search engines will bring this page up when people search for them.

Reply:
Ah! That makes perfect sense. I'll go back through and make sure the technologies are added back in everywhere.